misc/review/MidiServiceTransportPluginInfo.idl

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+namespace MIDI_MIDL_NAMESPACE
+{
+    MIDI_API_CONTRACT_v1
+    [default_interface]
+    runtimeclass MidiServiceTransportPluginInfo
+    {
+        Guid Id{ get; };
+
+        String Name{ get; };
+        String Mnemonic{ get; };
+        String Description{ get; };
+
+        String SmallImagePath{ get; };
+
+        String Author{ get; };
+
+        String Version{ get; };
```

Comment:
Is there a required format for the Version? Or is it free form?

Reply:
Free form. I spent some time looking for any sort of version type in winrt or Windows::Foundation and found nothing we could use. If you know of something, please LMK. I could roll my own, but that seemed like overkill given the use here.